src/ucp/core/ucp_ep.h

```diff
@@ -31,10 +31,12 @@ enum {
     UCP_EP_FLAG_CONNECT_REQ_QUEUED  = UCS_BIT(2), /* Connection request was queued */
     UCP_EP_FLAG_TAG_OFFLOAD_ENABLED = UCS_BIT(3), /* Endpoint uses tl offload for tag matching */
     UCP_EP_FLAG_FAILED              = UCS_BIT(4), /* EP is in failed state */
+    UCP_EP_FLAG_STREAM_IS_QUEUED    = UCS_BIT(5), /* EP is queued in stream list of worker */
 
-    UCP_EP_FLAG_CONNECT_REQ_SENT    = UCS_BIT(5), /* DEBUG: Connection request was sent */
-    UCP_EP_FLAG_CONNECT_REP_SENT    = UCS_BIT(6), /* DEBUG: Connection reply was sent */
-    UCP_EP_FLAG_CONNECT_ACK_SENT    = UCS_BIT(7)  /* DEBUG: Connection ACK was sent */
+    /* DEBUG bits */
```

Comment:
maybe hide it with `#if ENABLE_DEBUG_DATA` as well?

Reply:
I'll check if it's not producing a lot of changes... this is not a part of this change.

Reply:
`ucp_wireup_msg_progress` function uses these flags w/o any compile time checks. So, i don't think it should be part of this PR.

Reply:
ok